### ISSUE TITLE
__str__ in field_names

### ISSUE THREAD

**Issue by openmartin** [erdem/django-admino#4]
I found **str** in field_names site.py[141 line] cause unicode error and type error

So I add 

```
field_names.remove("__str__") 
```

I am using python 2.7, is this the problem?

**Comment by erdem** [comment]
Yes, that's should be a python 3 compatibility issue. We need to implement a fallback for this logic.
